docs/configuration.asciidoc

```diff
@@ -396,3 +396,13 @@ apm.init({
   }
 })
 ----
+
+
+[float]
+[[send-credentials]]
+==== `sendCredentials`
+
+* *Type:* Boolean
+* *Default:* `false`
+
+Instructs the agent to send credentials when making requests to the APM server.
```

Comment:
```suggestion


[float]
[[send-credentials]]
==== `sendCredentials`

* *Type:* Boolean
* *Default:* `false`

This allows the agent to send cookies when making requests to the APM server.
This is useful on scenarios where the APM server is behind reverse proxies that require requests to be authenticated.

NOTE: This feature requires changes on your server configuration. The server has to set `Access-Control-Allow-Credentials` header to `true` and `Access-Control-Allow-Origin` header to the origin doing the request rather than a wildcard.

```

Reply:
I recommend to update the documentation as suggested above (or something similar, in case you want to amend something)

On the screenshot below we can see how it would look like once the docs are updated:

<img width="843" alt="Screenshot 2022-06-13 at 13 58 01" src="https://user-images.githubusercontent.com/15065076/173348613-1e2db7d9-ab22-4a9c-a155-f02272284ed7.png">

Reply:
I took your comment and ran with it - I added a link to the CORS configuration section.

![image](https://user-images.githubusercontent.com/5742331/173384593-0ff9f538-8d23-47ef-a45a-b3e6ad2df941.png)